### PR TITLE
Cycle 28 revision 5.

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: build/schedview/Dockerfile [hunk 1]
```diff
@@ -4,16 +4,22 @@ ARG hub
...
 
 WORKDIR /home/saluser/schedview
 
 RUN /home/saluser/.checkout_repo.sh ${schedview}
 
+ARG bokeh
 RUN source /home/saluser/.setup_sal_env.sh && \
-    mamba install -y bokeh pytest-flake8 pytest-black selenium firefox geckodriver build
+    mamba install -y bokeh=${bokeh} \
+    pytest-flake8 \
```
Comment (mareuter) [inline build/schedview/Dockerfile:17]: Why are we installing packages for development into a deployment container?

File: build/rubintv-broadcaster/Dockerfile [hunk 2]
```diff
@@ -31,6 +31,10 @@ RUN mkdir -p /repos && \
 
 USER lsst
 
+RUN source ${WORKDIR}/loadLSST.bash && \
```
Comment (mareuter) [inline build/rubintv-broadcaster/Dockerfile:34]: Why not combine this with the one below? Saves an extra container layer.

**Review by mareuter — APPROVED** [review]
This looks fine. I think at some point we need to review our best practices for our Dockerfiles.

File: build/schedview/Dockerfile (hunk 1 above)
Comment (tribeiro) [reply build/schedview/Dockerfile:17]: I agree.. If I remember correctly there was something in this package that require these additional packages.. But this entire thing needs to be reworked once schedview becomes a conda package. So, in the end we should be doing `mamba install schedview`, but we are not there yet..

**Pull request merged**